docs/VehicleInfo/UnsubscribeVehicleData/index.md

```diff
@@ -59,6 +59,8 @@ HMI must send SubscribeVehicleData response only for ROOT level items.
 For OEM specific custom vehicle data items, `oemCustomDataType` will contain a type of OEM specific vehicle data (from schema), and `dataType` will be `VEHICLEDATA_OEM_CUSTOM_DATA`.  
 For vehicle data items from RPCSpec, `oemCustomDataType` will be omitted, and `dataType` will contain appropriate data type from `VehicleDataType` enum.
 
+In the case during data resumption with multiple application a subscription is already used by other applications, this means that the subscription is actual and SDL should not send unsubscribe requests to HMI.
```

Comment:
`and SDL should not send unsubscribe requests to HMI` the assumption of this is that all other data items of the same app that used the subscription

Reply:
correct.
@yang1070, would you like to have this rephrased like 
```suggestion
In the case during data resumption with multiple applications a subscription is already used by other applications, this means that the subscription is successfully restored and SDL should not send unsubscribe requests to HMI.
```

Reply:
In the case during data resumption with multiple applications, a subscription is already used by other applications, which have all their data successfully resumed, SDL should not send unsubscribe requests to HMI.

Reply:
updated in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/224/commits/0df9a2ae6494e5ec0438451d3cd0cfa0dd42b32a as agreed